### PR TITLE
Prettier public profile page

### PR DESCRIPTION
**Pull request by alexmiddeleer**

### CODE REVIEW

File: next/components/user-top-decks.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
```
Comment (alexmiddeleer) [inline next/components/user-top-decks.js:1]: To be honest this is basically a copy paste of user-newest-decks. Maybe I should DRY it up... thoughts?

Comment (lily-pytel) [reply next/components/user-top-decks.js:1]: Always!

Comment (lily-pytel) [reply next/components/user-top-decks.js:1]: It looks kind of the same as the homepage lists too so you can even further

Comment (alexmiddeleer) [reply next/components/user-top-decks.js:1]: Good points. I made some new components to DRY it up; lmk if they look alright.

File: next/components/view-more.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+
+function ViewMore({ url, count, limit }) {
```
Comment (lily-pytel) [inline next/components/view-more.js:3]: You can just put the limit here, since I believe `defaultProps` are going away

**Comment by alexmiddeleer** [comment]
> The code and profile page look great! I would just take out the underline in the view more link but it's whatever.
> image

Good catch on the underline. Might as well make it look like we're trying to match the design at least a bit, heh.

**Pull request merged**